### PR TITLE
Is1032/document ci workflow

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- [x] adds a README for the CI integration in osparc-services
- [x] adds links to microbadger for all images in the TOC

## Checklist

- [ ] I think the code is well written
- [x] I updated the table of contents (make toc)
- [ ] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: ops/diagram-ci.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+participant Dockerhub
+
+Master->>Feature: create feature1 branch on fork
+loop developing
+    Feature->>Feature: CI: build/test
+    Feature-->>Dockerhub: Deploy to personal registry
+end
+Feature-->>Master: Pull Request feature1
+Master->Master: CI: build/test
+Master->>Dockerhub: Deploy to itisfoundation repository
```
Comment (pcrespov) [inline ops/diagram-ci.md]: Above dockerhub is refered as *registry* and below as *repository*. Would be better to use one or the other. I propose  ``docker images repository`` (DIR)

File: ops/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Travis CI
+
+The Travis continuous integration tool is used.
+
+## Release workflow
+
```
Comment (pcrespov) [inline ops/README.md]: Don't we need to set some variables in travis so it has access to the personal docker image repository? Shouldn't you add that info in the doc under something like ``setting up your personal DIR``?

File: ops/diagram-ci.md (hunk 1 above)
Comment (sanderegg) [reply ops/diagram-ci.md]: itisfoundation is the repository, dockerhub the registry

File: ops/README.md (hunk 2 above)
Comment (sanderegg) [reply ops/README.md]: very right, that is now corrected

File: ops/diagram-ci.md (hunk 1 above)
Comment (pcrespov) [reply ops/diagram-ci.md]: Actually the guys of docker call it repository in dockerhub  :-) For that reason I wanted to be a bit more precise
We can add a small clarification as a footnote: e.g. ``The terms *repository* and *registry* above refer to VCS repository in github and docker images registry in dockerhub, resp.``

**Pull request merged**